doc/source/memory-management.rst

```diff
@@ -3,8 +3,57 @@ Memory Management
 
 This page describes how memory management works in Ray, and how you can set memory quotas to ensure memory-intensive applications run predictably and reliably.
 
-Overview
---------
+
+Summary
+-------
+
+You can set memory quotas to ensure your application runs predictably on any Ray cluster configuration. If you're not sure, you can start with a conservative default configuration like the following and see if any limits are hit.
+
+For Ray initialization on a single node, consider setting the following fields:
+
+.. code-block:: python
+
+  ray.init(
+      memory=2000 * 1024 * 1024,
+      object_store_memory=200 * 1024 * 1024,
+      driver_object_store_memory=100 * 1024 * 1024)
+
+For Ray usage on a cluster, consider setting the following fields on both the command line and in your Python script:
+
+.. code-block:: bash
+
+  # On the head node
+  ray start --head --redis-port=6379 \
+      --object-store-memory=200000000 \
```

Comment:
The memory limits here don't match in `ray.init()` above which is a bit confusing. Would stick with GiB over GB in both places.

Reply:
```suggestion
      --object-store-memory=200 * 1024 * 1024 \
```

Reply:
That's why I like standard lib/reference like this that new users can refer to:
https://bitmath.readthedocs.io/en/latest/classes.html

when they see:

`ray start --head --redis-port=6379 --object-store-memory=bitmath.MB(200)`